src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/cft/query/RoleAssignmentFilter.java

```diff
@@ -163,19 +173,21 @@ private static Predicate mapSecurityClassification(
                     SecurityClassification.PUBLIC
                 )
             );
-        } else {
+        } else if (classification.equals(Classification.PUBLIC)) {
```

Comment:
unit test for when !equal?

Reply:
I have to move that else part, classification cannot be other than private, public and restricted.